Formula/r/rustfmt.rb

```diff
@@ -17,6 +17,8 @@ class Rustfmt < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "0e93863d9f42ac0ec4bda61168758d82e2164a400ea80d929433c864728977ba"
   end
 
+  deprecate! date: "2024-09-20", because: "needs pre-built Rust libraries"
```

Comment:
Just delete this and add
```json
"rustfmt": "rust"
```
to `formula_renames.json`, I think.

Reply:
Only situation I wasn't sure is if there was any odd use case of `rustfmt` without brew `rust` that may break from this.

Otherwise, can merge formulae.

Reply:
Updated #191372 with merging formulae